server/Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'mongoid', '~> 4.0.0'
 gem 'symmetric-encryption'
 gem 'activesupport', '~> 4.1.14'
 gem 'faye-websocket', '~> 0.10.0'
+gem 'eventmachine', '>= 0.12.0'
```

Comment:
Okay, just randomly selected the same as faye-websocket is using. But let's remove that dependency as redundant(?))